### PR TITLE
fix student version

### PR DESCRIPTION
**Pull request by Gryfenfer97**
Fix student versions that were not detected if a non-student version exist

### CODE REVIEW

File: tests/test_path.py [hunk 1]
```diff
@@ -185,14 +233,34 @@ def test_inexistant_mechanical(mock_filesystem):
...
+        assert lowercase_available_ansys_installation == dict(
+            zip([202, 211, 231] + [-201, -211], lowercase_ansys_installation_paths)
+        )
+    else:
+        assert get_available_ansys_installations() == dict(
+            zip(
+                [202, 211, 231] + [-201, -211],
+                ANSYS_INSTALLATION_PATHS + ANSYS_STUDENT_INSTALLATION_PATHS,
+            )
+        )
```
Comment (germa89) [inline tests/test_path.py:253]: I would have break this tests in one for linux and another for windows.

**Review by germa89 — APPROVED** [review]
LGTM. Thank you for this @Gryfenfer97 !

But I just wonder why we weren't able to detect ansys student versions. I believe that was possible when this was part of PyMAPDL.

**Comment by Gryfenfer97** [comment]
> LGTM. Thank you for this @Gryfenfer97 !
> 
> But I just wonder why we weren't able to detect ansys student versions. I believe that was possible when this was part of PyMAPDL.

- For windows

It was because of [this comparison](https://github.com/ansys/ansys-tools-path/pull/66/files#diff-2578fa36965eb52a2a2edeef192d6f3902e611ef7916bc7a56b3e6bb4aac529eL85):
```python
if "student" in path_.lower() and os.path.exists(path_non_student):
    # add student version and normal version
else:
    awp_roots.append((ver, path_))
```

The problem was the case if there is a student version but the non student version doesn't exists. In that case we would add the student version as a normal version (without the minus for the version)

- For linux

[We were only checking if there is no normal version](https://github.com/ansys/ansys-tools-path/pull/66/files#diff-2578fa36965eb52a2a2edeef192d6f3902e611ef7916bc7a56b3e6bb4aac529eL85-L87) and even if there was indeed no normal version [we added the student version as a non student version
](https://github.com/ansys/ansys-tools-path/pull/66/files#diff-2578fa36965eb52a2a2edeef192d6f3902e611ef7916bc7a56b3e6bb4aac529eL139) (without the minus)

File: tests/test_path.py (hunk 1 above)
Comment (Gryfenfer97) [reply tests/test_path.py]: done

**Comment by germa89** [comment]
Got it! Good catch!

**Pull request merged**